browser/brave_wallet/brave_wallet_provider_delegate_impl.cc

```diff
@@ -88,7 +88,7 @@ void BraveWalletProviderDelegateImpl::OnConnectionError() {
 
 GURL BraveWalletProviderDelegateImpl::GetOrigin() const {
   auto* rfh = content::RenderFrameHost::FromID(host_id_);
-  return rfh ? rfh->GetLastCommittedURL().GetOrigin() : GURL();
+  return rfh ? rfh->GetLastCommittedURL().DeprecatedGetOriginAsURL() : GURL();
```

Comment:
should be `rfh->GetLastCommittedOrigin().GetURL()`.

Reply:
Missed that one, sorry. Fixed in https://github.com/brave/brave-core/commit/02855182fa0d19d311ec82c3ac847310c2899106